spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/command/SkipperCommands.java

```diff
@@ -265,4 +268,53 @@ public String upload(@ShellOption(help = "the package to be uploaded") String pa
 		PackageMetadata packageMetadata = skipperClient.upload(properties);
 		return "Package uploaded successfully:[" + packageMetadata.getName() + ":" + packageMetadata.getVersion() + "]";
 	}
+
+	@ShellMethod(key = "list", value = "List of releases with the latest revision on state deployed or failed")
+	public Table listReleases(
```

Comment:
Let's follow the convention that the method name should be same as command name.